src/chrome-launcher.ts

```diff
@@ -276,7 +275,7 @@ class Launcher {
   }
 
   // resolves when debugger is ready, rejects after 10 polls
-  private waitUntilReady() {
+  public waitUntilReady(): Promise<void> {
```

Comment:
no need for public, as this is inferred/default, and there is no need to set the return type of the function as it is also inferred.

Reply:
@samccone Fixed in e3c8ab8.